test/libethereum/StateTestsFiller/stCallCreateCallCodeTestFiller.json

```diff
@@ -618,6 +618,96 @@
         }
     },
 
+    "createNameRegistratorPreStore1NotEnoughGas": {
+        "env" : {
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6",
+            "currentNumber" : "0",
```

Comment:
this is only the filler, here we can use what we want :-), when creating the real tests, everything his hexified.